backend/app/resources/post.py

```diff
@@ -325,3 +325,42 @@ def delete(self, id):
                 return {"message": "Tag removed successfully."}, 204
             else:
                 return {"error": "No tag found with this id in post tags."}, 404
+
+def getOtherUsersPostsWithUserID(id):
```

Comment:
PEP8 requires function or method names to be in _snake_case_
[Refer here](https://peps.python.org/pep-0008/#function-and-variable-names)